_includes/neurips/2019/about.neurips.2019.es.html

```diff
@@ -16,28 +16,27 @@ <h4>Creando Oportunidades para LatinX en IA</h4>
+        <p>En general, los trabajos aceptados serán presentados en una sesión de pósters. Los autores aceptados para presentar recibiran si lo desean entrenamiento. Habrá un número reducido de trabajos que serán seleccionados para que sus autores hagan una presentación oral de 15 minutos. Las propuestas serán revisadas por voluntarios de la comunidad. Invitamos a que los autores participen como voluntarios para revisar las propuestas para LatinX en IA, indicándolo así durante el proceso de envío de sus resúmenes.</p>
 
         <h2>Guia de envios</h2>
-        <p>Todos los articulos enviados seran sometidos a una revision doble sin nombre de autor. Las presentaciones pueden tener hasta dos páginas, incluidas todas las figuras y tablas, con una página adicional para las referencias enviadas como un enlace a un PDF. Las presentaciones deben seguir el <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles">formato estandar </a> de la conferencia. Deben estar compuestas por tipos de letra de 11 puntos y deben tener al menos un margen de 1 pulgada alrededor. Las presentaciones que no sigan estas pautas corren el riesgo de ser rechazadas sin consideración de sus méritos. Las presentaciones deben indicar el problema de investigación, la motivación y la contribución técnica. Las presentaciones deben ser independientes e incluir todas las figuras, tablas y referencias.</p>
+        <p>Todos los artículos envíados serán sometidos a una evaluación de doble ciego (autores y evaluadores anónimos). Las presentaciones pueden tener hasta dos páginas, incluidas todas las figuras y tablas, con una página adicional para las referencias enviadas como un enlace a un PDF, siguiendo el <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles"> estilo estandard </a> de la conferencia (tambien puede utilizar este template de <a href="https://www.overleaf.com/latex/templates/neurips-2019/tprktwxmqmgk"> overleaf </a>). Las presentaciones deben estar en una sola columna. Deben estar compuestas por tipos de letra de 11 puntos y deben tener al menos un margen de 1 pulgada alrededor. Las presentaciones que no sigan estas pautas corren el riesgo de ser rechazadas sin consideración de sus méritos. Las presentaciones deben indicar el problema de investigación, la motivación y la contribución técnica. Las presentaciones deben ser independientes e incluir todas las figuras, tablas y referencias.</p>
         <p>El plazo para enviar el resumen tecnico finaliza a las 11:59 p.m. EST, el 13 de septiembre de 2019. Tenga en cuenta que no se ofrecerán extensiones para las presentaciones.</p>
 
-<a class="project-link" href="https://cmt3.research.microsoft.com/LXAINeurlPS2019">Envio del Resumen Tecnico</a>
+        <a class="project-link" href="https://cmt3.research.microsoft.com/LXAINeuriPS2019">Envio del Resumen Tecnico</a>
```

Comment:
The 'i' should be capitalized. This may have already been fixed in a previous pull request from @juancamilog 
Did you pull from the master branch prior to making these changes?